Makefile

```diff
@@ -42,7 +42,7 @@ devel-install:
 dist-gzip: NODE_ENV=production
 dist-gzip: all $(PACKAGE_NAME).spec
 	mkdir -p $(PACKAGE_NAME)-$(VERSION)
-	cp -r public/ LICENSE.txt README.md $(PACKAGE_NAME).spec io.weldr.cockpit-composer.metainfo.xml $(PACKAGE_NAME)-$(VERSION)
+	cp -r public/dist test LICENSE.txt README.md $(PACKAGE_NAME).spec io.weldr.cockpit-composer.metainfo.xml $(PACKAGE_NAME)-$(VERSION)
```

Comment:
IMHO we should still ship public/ for license reasons (preferred form of modification). Also, it's just 30 kB, hardly worth optimizing for. :wink: